test/benchmark/Mesa/WolfSheep/benchmark.py

```diff
@@ -17,7 +17,6 @@
 wolfsheepmodel = WolfSheep()
 """
 
-for a in range(0, 10):
-    tt = timeit.timeit('wolfsheepmodel.run_model()', setup=setup, number=1)
-    print(tt)
-
+tt = timeit.Timer('wolfsheepmodel.run_model()', setup=setup)
+a = min(tt.repeat(100, 1))
```

Comment:
I was having some real issues with the documentation here. 
- What is the default unit of `timeit`?
- `timeit.timeit` does repeats internally, defaults to 100 runs. Does `timeit.Timer` not do that? Then you just control it by the `tt.repeat` clause?

Reply:
Timer is just an object but doesn't run. We can explicitly control the number of repeats.